C-Plus-Plus/dp/bellman_ford.cpp

```diff
@@ -0,0 +1,128 @@
+#include<bits/stdc++.h>
+using namespace std;
+#define MAX 10
```

Comment:
Don't define max
Take input from the user
Thanks!

Reply:
done @siddharth25pandey Kindly review it